components/share-modal/share-modal.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+// @ts-ignore
+import styles from "./share-modal.module.scss";
+import SVG from "react-inlinesvg";
+// @ts-ignore
+import facebookIcon from "../../public/images/social/facebook.svg";
```

Comment:
Do the import directly from /images like so:
import facebookIcon from '/images/social/facebook.sgv'

The same goes for all the icons.